docs/src/1_Mechanics/04_Powers.yaml

```diff
@@ -918,4 +918,202 @@ Amplify Ally:
     PP: 0
     Description: Move the chess pieces on your board.
     Mechanic: Select one ally. They can immediately move up to 4 spaces and take one Action.  
-    
\ No newline at end of file
+
+Craft Light:
+    Category:
+        - Magic
+    Description: Your skills let you conjure illusory effects.
+    Mechanic: Make a DR 3 Craft Check. On a Success you create an 8 space radiance of light for 15 minutes. On a Failure, the light only lasts for 1 round. You can create a temporary illusory object or infuse an existing object with this property. It can be covered and dimmed. Using a Minor action allows you to extinguish a light. If the light is more than 200 spaces away from you, it disappears
```

Comment:
```suggestion
    Mechanic: Make a DR 3 Craft Check. On a Success you create an Aura 6 space radiance of light for 15 minutes. On a Failure, the light only lasts for 1 round. You can create a temporary illusory object or infuse an existing object with this property. It can be covered and dimmed. Using a Minor action allows you to extinguish a light. If the light is more than 100 spaces away from you, it disappears
```